README.md

```diff
@@ -2,8 +2,15 @@
 
 ### SDP - spring semester 2021
 
+## Build
 [![Build Status](https://api.cirrus-ci.com/github/movemeet/movemeet.svg)](https://cirrus-ci.com/github/movemeet/movemeet)
 
+## Maintainability
+[![Maintainability](https://api.codeclimate.com/v1/badges/23a7b536f058d3bfda16/maintainability)](https://codeclimate.com/github/movemeet/movemeet/maintainability)
+
```

Comment:
It will look marvelous after it's merged.

Reply:
Indeed, that's the easy part ^^